apps/ident-service/src/main/java/no/nav/registre/testnorge/identservice/service/IdentAppService.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.registre.testnorge.identservice.service;
+
+import lombok.SneakyThrows;
+import no.nav.registre.testnorge.identservice.testdata.FiltrerPaaIdenterTilgjengeligIMiljo;
+import no.nav.registre.testnorge.identservice.testdata.response.IdentMedStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class IdentAppService {
+
+    @Autowired
+    private FiltrerPaaIdenterTilgjengeligIMiljo filtrerPaaIdenterTilgjengeligIMiljo;
```

Comment:
Ikke autowire dirkte i på variablen(Umulig å teste). 

Hvis det er en contructor i spring så vil den automatik bli `@Autowired`.

Noe som betur at vi kan bruke Lombok sin `@RequiredArgsConstructor` til å automatisk Autowireder. Så lenge variablene er final (da blir den required). 

```java
@Service
@RequiredArgsConstructor
public class IdentAppService {
    private final FiltrerPaaIdenterTilgjengeligIMiljo filtrerPaaIdenterTilgjengeligIMiljo;

    @SneakyThrows
    public ResponseEntity<IdentMedStatus> finnLedigeIdenter(String ident) {

        IdentMedStatus identMedStatus = filtrerPaaIdenterTilgjengeligIMiljo.filtrerPaaIdenter(ident);

        return ResponseEntity.ok().body(identMedStatus);
    }
}
```

Reply:
Ahh okay, takk for tips! 😄